### ISSUE TITLE
For dc_rights field can we move to using Oregon Digital's rights stuff?

### ISSUE THREAD

**Issue by mejackreed** [geoblacklight/geoblacklight-schema#52]
http://oregondigital.org/copyright

**Comment by kimdurante** [comment]
It might be worth implementing a rights field for both access and use/reproduction. I think we've done this for Stanford materials already.
Perhaps using:
'dct:accessRights' to manage access to the data
and
'dct:license' to contain the license types or use restrictions. Perhaps we could use the URIs to the specific licenses as element values, when available.

**Comment by andrewbattista** [comment]
I agree with Kim. I can already see instances where we would want to make it clear with the exact usage rights are for each piece of data, especially when we are collecting shapefiles that are derivatives created with proprietary data sources.

**Comment by mejackreed** [comment]
Also see #76

**Comment by andrewbattista** [comment]
I'm adding this for consideration as we revisit the Schema for the upcoming 2017 Sprint
